src/App/Pages/Accounts/LoginPasswordlessRequestViewModel.cs

```diff
@@ -0,0 +1,196 @@
+            _logger = ServiceContainer.Resolve<ILogger>();
+
+            PageTitle = AppResources.LogInWithAnotherDevice;
+
+            CreatePasswordlessLoginCommand = new AsyncCommand(() => Device.InvokeOnMainThreadAsync(CreatePasswordlessLoginAsync),
+                onException: ex => HandleException(ex),
+                allowsMultipleExecutions: false);
+        }
+
+        public Action StartTwoFactorAction { get; set; }
+        public Action LogInSuccessAction { get; set; }
+        public Action UpdateTempPasswordAction { get; set; }
+        public Action CloseAction { get; set; }
+
+        public ICommand CreatePasswordlessLoginCommand { get; }
+
+        public string FingerprintPhrase
+        {
+            get => _fingerprintPhrase;
+            set => SetProperty(ref _fingerprintPhrase, value);
+        }
+
+        public string Email
+        {
+            get => _email;
+            set => SetProperty(ref _email, value);
+        }
+
+        public void StartCheckLoginRequestStatus()
+        {
+            try
+            {
+                _checkLoginRequestStatusCts?.Cancel();
+                _checkLoginRequestStatusCts = new CancellationTokenSource();
+                _checkLoginRequestStatusTask = new TimerTask(_logger, CheckLoginRequestStatus, _checkLoginRequestStatusCts).RunPeriodic(TimeSpan.FromSeconds(REQUEST_TIME_UPDATE_PERIOD_IN_SECONDS));
+            }
+            catch (Exception ex)
+            {
+                _logger.Exception(ex);
+            }
+        }
+
+        public void StopCheckLoginRequestStatus()
+        {
+            try
+            {
+                _checkLoginRequestStatusCts?.Cancel();
+                _checkLoginRequestStatusCts?.Dispose();
+                _checkLoginRequestStatusCts = null;
+            }
+            catch (Exception ex)
+            {
+                _logger.Exception(ex);
+            }
+        }
+
+        private async Task CheckLoginRequestStatus()
+        {
+            if (string.IsNullOrEmpty(_requestId) || string.IsNullOrEmpty(_requestAccessCode))
+            {
+                return;
+            }
+
+            try
+            {
+                var response = await _authService.GetPasswordlessLoginResponseAsync(_requestId, _requestAccessCode);
+
+                if (!response.RequestApproved)
+                {
+                    return;
+                }
+
+                StopCheckLoginRequestStatus();
+
+                var authResult = await _authService.LogInPasswordlessAsync(Email, _requestAccessCode, _requestId, _requestKeyPair.Item2, response.Key, response.MasterPasswordHash);
+                await AppHelpers.ResetInvalidUnlockAttemptsAsync();
+
+                if (!await HandleCaptchaAsync(authResult.CaptchaSiteKey, authResult.CaptchaNeeded, CheckLoginRequestStatus))
+                {
+                    return;
+                }
+
+                await _deviceActionService.HideLoadingAsync();
```

Comment:
Not quite sure where the loading is starting, but if you hide it here you need to hide it as well before the `return` in the if